projects/osquery/project.yaml

```diff
@@ -1,7 +1,10 @@
 homepage: "https://osquery.io"
-primary_contact: "theopolis@osquery.io"
+primary_contact: "osquery@osquery.io"
 auto_ccs:
-  - "osquery@osquery.io"
+  - "theopolis@osquery.io"
+  - "seph@kolide.co"
+  - "seph@directiopnless.org"
```

Comment:
directiopnless.org - typo ??

Reply:
Indeed. You might think I can type my own email address. Thank you